Unable to run backstage with the snyk plugin using the latest cli (webpack 5)

I'm getting errors like the following when I try to run backstage with the snyk plugin:

```
[0] ERROR in ../../node_modules/@snyk/dep-graph/dist/legacy/index.js 4:15-32
[0] Module not found: Error: Can't resolve 'crypto' in '/Users/iainbillett/code/backstage/node_modules/@snyk/dep-graph/dist/legacy'
[0] 
[0] BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
[0] This is no longer the case. Verify if you need this module and configure a polyfill for it.
[0] 
[0] If you want to include a polyfill, you need to:
[0]     - add a fallback 'resolve.fallback: { "crypto": require.resolve("crypto-browserify") }'
[0]     - install 'crypto-browserify'
[0] If you don't want to include a polyfill, you can use an empty module like this:
[0]     resolve.fallback: { "crypto": false }
```

## Expected Behavior

I can use the snyk plugin ([backstage-plugin-snyk](https://github.com/snyk-tech-services/backstage-plugin-snyk))

## Current Behavior

When I start the app (e.g. `yarn dev`) webpack fails with the error above. 

## Possible Solution

I've tested setting resolve.fallback.crypto = false in packages/cli/src/lib/bundler/config.ts but presumably this could cause a runtime issue. It does compile though. 

## Steps to Reproduce

* Create a backstage instance (or upgrade an existing one) e.g. npx @backstage/create-app
* Follow instructions to install the Snyk plugin
* run yarn dev

## Context

I've made an [issue](https://github.com/snyk-tech-services/backstage-plugin-snyk/issues/29) with snyk also.

--- a/.changeset/neat-jars-rule.md
+++ b/.changeset/neat-jars-rule.md
@@ -1,0 +1,5 @@
+---
+'@backstage/cli': patch
+---
+
+Reintroduce Node.js shims that were removed in the Webpack 5 migration.

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -85,6 +85,7 @@
     "json-schema": "^0.3.0",
     "lodash": "^4.17.19",
     "mini-css-extract-plugin": "^1.4.1",
+    "node-libs-browser": "^2.2.1",
     "ora": "^5.3.0",
     "postcss": "^8.1.0",
     "process": "^0.11.10",

--- a/packages/cli/src/lib/bundler/config.ts
+++ b/packages/cli/src/lib/bundler/config.ts
@@ -32,6 +32,7 @@ import { BundlingOptions, BackendBundlingOptions, LernaPackage } from './types';
 import { version } from '../../lib/version';
 import { paths as cliPaths } from '../../lib/paths';
 import { runPlain } from '../run';
+import pickBy from 'lodash/pickBy';
 
 export function resolveBaseUrl(config: Config): URL {
   const baseUrl = config.getString('app.baseUrl');
@@ -177,6 +178,7 @@ export async function createConfig(
       extensions: ['.ts', '.tsx', '.mjs', '.js', '.jsx'],
       mainFields: ['browser', 'module', 'main'],
       fallback: {
+        ...pickBy(require('node-libs-browser')),
         module: false,
         dgram: false,
         dns: false,

